python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -86,9 +86,12 @@ def __deepcopy__(self, memo=None):
         cls = self.__class__
         obj = cls()
         this_dict = copy(self.__dict__)
-        for elem in this_dict.keys():
+        global do_not_copy
+        def internal_deepcopyMethod(elem):
             if elem not in do_not_copy:
-                this_dict[elem] = deepcopy(this_dict[elem], memo)  # FIXED
+                this_dict[elem] = deepcopy(this_dict[elem], memo)
+        for k in this_dict.keys():
+            internal_deepcopyMethod(k)
```

Comment:
code -> code with map -> code without map with bugs -> code without map working so I need to move back to the beginning of code in places where it makes sense.